src/desktop/assets/artists_artworks.coffee

```diff
@@ -1,9 +1,6 @@
 require('backbone').$ = $
 
 routes =
-  '/artist/.*': ->
-    require('../apps/artist/client/index.coffee').init()
-
```

Comment:
Now taken care of via https://github.com/artsy/force/pull/3024/files#diff-24d93602cc2b0212be6294f5249cf65cR2, yeah?

Reply:
yup exactly